includes/class-admin-settings.php

```diff
@@ -502,10 +532,10 @@ public function add_settings_field_callback( $args = array() ) {
 					<?php
 					foreach ( $group_options as $group_option ) {
 						?>
-							<option 
-								data-api-key-url="<?php echo esc_html( $group_option['api-key-url'] ?? '' ); ?>" 
-								data-require-api-key="<?php echo esc_html( $group_option['require-api-key'] ?? 'false' ); ?>" 
-								value="<?php echo esc_attr( $group_option['value'] ?? '' ); ?>" 
+							<option
+								data-api-key-url="<?php echo esc_html( $group_option['api-key-url'] ?? '' ); ?>"
+								data-require-api-key="<?php echo esc_html( $group_option['require-api-key'] ?? 'false' ); ?>"
+								value="<?php echo esc_attr( $group_option['value'] ?? '' ); ?>"
```

Comment:
🤔 What change am I not seeing?

Reply:
VSCode removed the extra space at the end of some of the lines automatically.

Reply:
Exactly 😂